components/movieDetails/movieDetails.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
```

Comment:
component name and file name must be the same. 
MovieDetails => MovieDetailes.jsx